dcpy/models/lifecycle/ingest.py

```diff
@@ -77,12 +78,10 @@ class Ingestion(SortedSerializedBase):
     processing_steps: list[ProcessingStep] = []
 
 
-class Column(SortedSerializedBase):
-    id: str
-    data_type: Literal[
-        "text", "integer", "decimal", "geometry", "bool", "date", "datetime"
-    ]
-    description: str | None = None
+class Column(BaseColumn):
+    _head_sort_order = ["id", "data_type", "description"]
+
+    data_type: COLUMN_TYPES  # override BaseColumn `data_type` to be required field
```

Comment:
In the `BaseColumn`, data_type can be of `COLUMN_TYPES` or `None`. Making it a required field for ingest.

Reply:
I had description as part of these columns?? I had no idea lol. This note of yours makes sense though - I don't think the code comment is needed